SyncedSideBar.py

```diff
@@ -142,11 +145,22 @@ def on_activated(self, view):
 
     # Sublime text v3 window command listener, safe to include unconditionally as it's simply ignored by v2.
     # Eventually, v3 support below 3098 will be dropped and this can be deleted.
-    def on_window_command(self, window, command_name, args):
-        if command_name == 'toggle_side_bar':
+    def on_window_command(self, window, command, args):
+        if command == 'toggle_side_bar':
             global sidebarVisible
             sidebarVisible = not sidebarVisible
 
+        # This is needed for the `on_post_window_command` to work
+        if command == 'reveal_in_side_bar':
+            self.last_view = window.active_view()
+
+
+    # Back to the file window after `reveal_in_side_bar` command
+    def on_post_window_command(self, window, command, args):
+        if command == 'reveal_in_side_bar':
+            if self.packageSettings.get('focus-on-view'):
```

Comment:
It's definitely changed in ST4, but you're right it might be a bug. Mostly, since having this on doesn't seem to impact ST3, I'd rather not make it an option.